docker-local-mojaloop/postman/OSS-New-Deployment-FSP-Setup.postman_collection.json

```diff
@@ -0,0 +1,7817 @@
+{
```

Comment:
it is almost impossible to review postman json collection

Reply:
yea...postman...this will get address in the future.